<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editBook</class>
 <widget class="QMainWindow" name="editBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>762</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1296</width>
    <height>762</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1296</width>
    <height>762</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Book</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/mainicon.png</normaloff>:/icons/icons/mainicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget
{
	border-image: url(:/new/photos/editBook.jpg);
}

QPushButton
{
	color: rgb(0, 122, 0);
	background-color: rgb(85, 255, 0);
	font: 24pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
    border-radius: 10px;
}
QPushButton:hover
{
	background-color: rgb(0, 122, 0);
	color:  rgb(85, 255, 0);
}
QLabel
{
	background-color: rgb(147, 221, 0);
	border-radius: 15px;
}
QLineEdit
{
	
	background-color: rgba(255, 255, 127, 0.9);
	border-radius: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dashButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;</string>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>1232</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Close the app</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/icons/error.png</normaloff>:/icons/icons/error.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Back to main menu</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/icons/previous.png</normaloff>:/icons/icons/previous.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ISBN:</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Number of Availables:</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="publisherEdit">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>430</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="ISBNEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>000-0-00-000000-0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="subjectEdit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>550</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>430</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Publisher:</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPage">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 127, 0.9);
	border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinAvail">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 127, 0.9);
	border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Book Subject:</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Book Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="authorEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>430</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Number of Pages:</string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Language: </string>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="languageEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>647</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;</string>
    </property>
    <property name="text">
     <string>Save changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Choose another book</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
